src/components/AppLayout/index.tsx

```diff
@@ -91,6 +93,14 @@ const Layout: React.FC<Props> = ({
 
   const closeMobileNotSupported = () => setMobileNotSupportedClosed(true)
 
+  const settingsBaseRoute = safeAddress
+    ? generatePath(SAFE_ROUTES.SETTINGS_BASE_ROUTE, {
+        safeAddress,
+      })
+    : undefined
+  const footerRegex = new RegExp(`(${WELCOME_ADDRESS}|${settingsBaseRoute})`)
+  const hasFooter = footerRegex.test(useLocation().pathname)
```

Comment:
It'll work but I personally don't like to send hooks as arguments. What are your thoughts @katspaugh?

Reply:
I know what you mean, probably this style would be clearer:
```
const { pathname } = useLocation()
const hasFooter = footerRegex.test(pathname)
```